notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sns.distplot(df[df["rok_zalozenia_cat"] &gt;= 1900]["rok_zalozenia_cat"]);
</pre><p>Dużo mamy tych szkół poniżej 1900 roku? Może nie ma potrzeby ucinać tego długiego ogona?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>